experiment/EASR/config.txt
2020-11-18-22:29:16

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:22

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:22

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:22

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:22

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:29

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:29

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:29

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:29

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_best.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:45

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:45

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:45

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:45

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:53

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:58

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:58

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:58

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:29:58

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:30:05

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:30:05

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:30:05

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:30:20

act: relu
pre_train: X:\EASR\EASR_model_x4\EASR_CBAM_RIR_x4\model\model_latest.pt
extend: .
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 8
n_resblocks: 16
reduction: 16
batch_size: 16
input_size: 48
scale: 4
dir_train: G:\DIV2K\
dir_test: G:\DIV2K\Test\
dir_demo: ./Test_real_small/
data_name: DIV2K_8
demo_name: Real_16
test_name: ['Set5', 'Set14', 'B100', 'Urban100']
rgb_range: 1
n_colors: 3
test_only: True
demo: False
cpu: False
n_GPUs: 1
loss: 1*L1
self_ensemble: False
chop: False
gan_k: 1
test_every: 1000
epochs: 300
local_rank: -1
seed: 1
reset: False
model_name: EASR
lr: 0.0001
decay: 200-400-600-800-1000
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
save: EASR
load: 
resume: 0
save_models: False
print_every: 100
save_results: False
save_gt: False

2020-11-18-22:30:20

act: relu

config.py

# coding :utf-8
import argparse
# DATA
DIR_Train = 'G:\\DIV2K\\'
DIR_Test = 'G:\DIV2K\Test\\'
DIR_DEMO = './Test_real_small/'
SCALE = 4
BATCH_SIZE = 16
INPUT_SIZE = 48
DATA_NAME ='DIV2K_8'
DEMO_NAME = 'Real_16'
TEST_NAME = ['Set5', 'Set14', 'B100', 'Urban100']

# TRAIN and TEST
TEST_ONLY = False
CPU = False
N_GPUs = 1
LOSS = '1*L1'
PRECISION = 'single'
# MODEL
MODEL_NAME = 'EASR'

parser = argparse.ArgumentParser(description="SuperResolution")

# Model_1 EDSR
parser.add_argument('--act', type=str, default='relu',
                    help='activation function')
parser.add_argument('--pre_train', type=str, default='pretrained_models\model_x4.pt',
                    help='pre-trained model directory')
parser.add_argument('--extend', type=str, default='.',
                    help='pre-trained model directory')

parser.add_argument('--n_feats', type=int, default=64,
                    help='number of feature maps')
parser.add_argument('--res_scale', type=float, default=1,
                    help='residual scaling')
parser.add_argument('--shift_mean', default=True,
                    help='subtract pixel mean from the input')
parser.add_argument('--dilation', action='store_true',
                    help='use dilated convolution')
parser.add_argument('--precision', type=str, default='single',
                    choices=('single', 'half'),
                    help='FP precision for test (single | half)')

# Model_2 RDN_DW

parser.add_argument('--G0', type=int, default=64,
                    help='default number of filters. (Use in RDN)')
parser.add_argument('--RDNkSize', type=int, default=3,
                    help='default kernel size. (Use in RDN)')
parser.add_argument('--RDNconfig', type=str, default='B',
                    help='parameters config of RDN. (Use in RDN)')

# Model_3 RIR
parser.add_argument('--n_resgroups', type=int, default=8,
                    help='number of residual blocks')
parser.add_argument('--n_resblocks', type=int, default=16,
                    help='number of residual blocks')
parser.add_argument('--reduction', type=int, default=16,
                    help='number of residual blocks')

# DATA configuration
parser.add_argument("--batch-size", type=int, default=BATCH_SIZE,
                    help="Number of images a batch")
parser.add_argument("--input-size", type=int, default=INPUT_SIZE,
                    help="Size of training input.")
parser.add_argument("--scale", type=int, default=SCALE,
                    help="Scale factor")
parser.add_argument("--dir-train", type=str, default=DIR_Train,
                    help="Dir of LR images")
parser.add_argument("--dir-test", type=str, default=DIR_Test,
                    help="Dir of HR images")
parser.add_argument("--dir-demo", type=str, default=DIR_DEMO,
                    help="Dir of HR images")
parser.add_argument("--data-name", type=str, default=DATA_NAME,
                    help="name of training dataset")
parser.add_argument("--demo-name", type=str, default=DEMO_NAME,
                    help="name of training dataset")
parser.add_argument("--test-name", type=list, default=TEST_NAME,
                    help="name of test dataset")
parser.add_argument('--rgb_range', type=int, default=1,
                    help='maximum value of RGB')
parser.add_argument('--n_colors', type=int, default=3,
                    help='number of color channels to use')


# Training and Testing configuration
parser.add_argument("--test-only", type=bool, default=False,
                    help="Test only or not")
parser.add_argument("--demo", type=bool, default=False,
                    help="Test only or not")
parser.add_argument("--cpu", type=bool, default=CPU,
                    help="use coy only")
parser.add_argument('--n_GPUs', type=int, default=1,
                    help='number of GPUs')
parser.add_argument('--loss', type=str, default=LOSS,
                    help='loss function configuration')
parser.add_argument('--self_ensemble', default=False,
                    help='use self-ensemble method for test')
parser.add_argument('--chop', default=False,
                    help='use chop method for test')
parser.add_argument('--gan_k', type=int, default=1,
                    help='k value for adversarial loss')
parser.add_argument('--test_every', type=int, default=1000,
                    help='do test per every N batches')
parser.add_argument('--epochs', type=int, default=300,
                    help='number of epochs to train')

parser.add_argument("--local_rank", type=int, default=-1)

parser.add_argument('--seed', type=int, default=1,
                    help='random seed')
parser.add_argument('--reset', action='store_true',
                    help='reset the training')
# Model configuration
parser.add_argument("--model-name", type=str, default=MODEL_NAME,
                    help="name of the model")

# Optimizer
parser.add_argument('--lr', type=float, default=0.0001,
                        help='learning rate')
parser.add_argument('--decay', type=str, default='200-400-600-800-1000',
                    help='learning rate decay type')

parser.add_argument('--gamma', type=float, default=0.5,
                    help='learning rate decay factor for step decay')
parser.add_argument('--optimizer', default='ADAM',
                    choices=('SGD', 'ADAM', 'RMSprop'),
                    help='optimizer to use (SGD | ADAM | RMSprop)')
parser.add_argument('--momentum', type=float, default=0.9,
                    help='SGD momentum')
parser.add_argument('--betas', type=tuple, default=(0.9, 0.999),
                    help='ADAM beta')
parser.add_argument('--epsilon', type=float, default=1e-8,
                    help='ADAM epsilon for numerical stability')
parser.add_argument('--weight_decay', type=float, default=0,
                    help='weight decay')
parser.add_argument('--gclip', type=float, default=0,
                    help='gradient clipping threshold (0 = no clipping)')

# Log specifications
parser.add_argument('--save', type=str, default='EASR',
                    help='file name to save')
parser.add_argument('--load', type=str, default='',
                    help='file name to load')
parser.add_argument('--resume', type=int, default=0,
                    help='resume from specific checkpoint')
parser.add_argument('--save_models', action='store_true',
                    help='save all intermediate models')
parser.add_argument('--print_every', type=int, default=100,
                    help='how many batches to wait before logging training status')
parser.add_argument('--save_results', default=False,
                    help='save output results')
parser.add_argument('--save_gt', action='store_true',
                    help='save low-resolution and high-resolution images together')


args = parser.parse_args()


